### PR TITLE
Changed outLFN and allowNonProductionCMSSW config parameter name.

### PR DESCRIPTION
**Pull request by nizamyusli**
Fixed https://github.com/dmwm/CRABClient/issues/4438
Fixed https://github.com/dmwm/CRABClient/issues/4219

### CODE REVIEW

File: src/python/CRABClient/ClientMapping.py [hunk 1]
```diff
@@ -67,15 +67,17 @@
...
+    'General.transferOutput'          : 'General.transferOutputs',
+    'General.saveLogs'                : 'General.transferLogs',
+    'Data.outlfn'                     : 'Data.outLFN',
+    'Data.outLFN'                     : 'Data.outLFNDirBase'
+    'Data.dbsUrl'                     : 'Data.inputDBS',
+    'Data.publishDbsUrl'              : 'Data.publishDBS',
+    'Data.userInputFile'              : 'Data.userInputFiles',
+    'JobType.numcores'                : 'JobType.numCores',
+    'JobType.maxmemory'               : 'JobType.maxMemoryMB',
+    'JobType.maxjobruntime'           : 'JobType.maxJobRuntimeMin'
```
Comment (AndresTanasijczuk) [inline src/python/CRABClient/ClientMapping.py]: a comma (,) is missing at the end of this line

File: src/python/CRABClient/ClientMapping.py [hunk 2]
```diff
@@ -67,15 +67,17 @@
...
-    'Data.outlfn'            : 'Data.outLFN',
-    'Data.dbsUrl'            : 'Data.inputDBS',
-    'Data.publishDbsUrl'     : 'Data.publishDBS',
-    'Data.userInputFile'     : 'Data.userInputFiles',
-    'JobType.numcores'       : 'JobType.numCores',
-    'JobType.maxmemory'      : 'JobType.maxMemoryMB',
-    'JobType.maxjobruntime'  : 'JobType.maxJobRuntimeMin'
+    'General.transferOutput'          : 'General.transferOutputs',
+    'General.saveLogs'                : 'General.transferLogs',
+    'Data.outlfn'                     : 'Data.outLFN',
```
Comment (AndresTanasijczuk) [inline src/python/CRABClient/ClientMapping.py]: New line 72 should be
'Data.outlfn' : 'Data.outLFNDirBase',

File: src/python/CRABClient/ClientMapping.py [hunk 3]
```diff
@@ -67,15 +67,17 @@
...
+    'Data.outLFN'                     : 'Data.outLFNDirBase'
+    'Data.dbsUrl'                     : 'Data.inputDBS',
+    'Data.publishDbsUrl'              : 'Data.publishDBS',
+    'Data.userInputFile'              : 'Data.userInputFiles',
+    'JobType.numcores'                : 'JobType.numCores',
+    'JobType.maxmemory'               : 'JobType.maxMemoryMB',
+    'JobType.maxjobruntime'           : 'JobType.maxJobRuntimeMin'
+    'JobType.allowNonProductionCMSSW' : 'JobType.allowUndistributedCMSSW'
```
Comment (ericvaandering) [inline src/python/CRABClient/ClientMapping.py]: In fact, please put a comma at the end of this line too. That way if you add something to the end of the list, you don't have to add a comma generating a spurious diff. IIRC, this works in most languages but Javascript.

Comment (ericvaandering) [inline src/python/CRABClient/ClientMapping.py]: On the other hand, current guidelines generally say that you should avoid adding all these extra spaces just to get things to align nicely. Again, it's more difficult to see what you are actually changing here as opposed to just spacing out differently. And it's a waste of time.

Comment (AndresTanasijczuk) [inline src/python/CRABClient/ClientMapping.py]: +1

**Comment by mmascher** [comment]
About `allowUndistributedCMSSW`, has somebody checked it is what we are doing in the server? Or are we only allowing the production releases there? I should know, I wrote the code but I do not remember :)

I can check if nobody checked.

**Comment by AndresTanasijczuk** [comment]
I have checked. See https://github.com/dmwm/CRABClient/issues/4219

**Pull request merged**